INSTALL.rst

```diff
@@ -217,7 +217,7 @@ libSplash
 HDF5
 """"
 - 1.8.6+
-- standard shared version (no c++, enable parallel)
+- standard shared version (no C++, enable parallel)
```

Comment:
I tried to use keywords here, but this is ok.
The user does *not* need the partially implemented C++ APIs (`--enable-cxx`) but parallel APIs (`--enable-parallel`).